lib/utils/express.ts

```diff
@@ -0,0 +1,36 @@
+import * as express from "express";
+import * as helmet from "helmet";
+import * as csp from "helmet-csp";
+import * as referrerPolicy from "referrer-policy";
+
+export function secureExpressApp(app: express.Express): void {
```

Comment:
this needs a short comment explaining what it does

Reply:
ok